### PR TITLE
MAT-5340: remove success message, update test

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-5340](https://jira.cms.gov/browse/MAT-5340)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is in to the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

**Review by sb-cecilialiu — APPROVED** [review]

File: src/AceEditor/madie-ace-editor.tsx [hunk 1]
```diff
@@ -293,7 +293,7 @@ const MadieAceEditor = ({
         <span>{editorAnnotations.length} issues were found with CQL...</span>
       );
     } else {
-      return <span>Parsing complete, CQL is valid</span>;
+      return <span data-testId="success-debounce"></span>;
```
Comment (adongare) [inline src/AceEditor/madie-ace-editor.tsx:296]: even though the scope of the story is to remove the success message, I think we do not need the whole footerdiv or this method. what do you think?

Comment (mcmcphillips) [reply src/AceEditor/madie-ace-editor.tsx:296]: What would be an alternative solution? The span is still present, but does not hold any text

Comment (adongare) [reply src/AceEditor/madie-ace-editor.tsx:296]: we already have an alternative. we display success/errors/warnings in the gutter and at the top of the editor, the whole footer seems to be redundant.

Comment (mcmcphillips) [reply src/AceEditor/madie-ace-editor.tsx:296]: I'll suggest we completely remove this gutter in favor of consolidated alerts

**Pull request merged**